src/style.css

```
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar {
+  box-sizing: border-box;
+  width: 100%;
+  max-height: 70px;
+  padding-left: 62px;
+  line-height: 70px;
+  font-size: inherit;
+  font-weight: 400;
+  font-family: "Avenir", sans-serif;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search-bar:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search-bar::placeholder {
+  margin-left: 25px;
+  border-radius: 4px;
+  text-shadow: none;
+}
+
+.search-bar:focus {
+  outline: none;
+  font-weight: 900;
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-form__icon {
+  position: relative;
+}
+
+.search-form__icon::after {
+  content: "";
+  position: absolute;
+  left: 26px;
+  bottom: 50%;
+  transform: translateY(50%);
+  width: 19px;
+  height: 19px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+}
+
+/* ------------data-qa="small"----------- */
+
+.search-form[data-qa="small"] {
+  font-size: 14px;
+}
+
+.search-form[data-qa="small"] .search-bar {
```

```suggestion
search-bar--small {
```
better to make modifier for small form and change properties which different from big form.
same for other elements